prometheus/monitoring-grafana-dashboard/README.md

```diff
@@ -0,0 +1,69 @@
+# Create a Grafana dashboard
+
+Follow these sections to create the Gauge dashboard type for the `cpu_temperature_celsius` metric.
+
+## Prerequisites
+
+* Kyma as the target deployment environment.
+* Deployed custom kube-prometheus-stack as described in the [prometheus](../) example.
+* Deployed [custom-metrics example](../monitoring-custom-metrics).
+
+## Create the dashboard
+
+1. [Access Grafana](../README.md#verify-the-installation).
+2. Add a new dashboard with a new panel.
+3. For your new query, select **Prometheus** from the data source selector.
+4. Pick the `cpu_temperature_celsius` metric.
+5. To retrieve the latest metric value on demand, activate the **Instant** switch.
```

Comment:
Would be really nice to specify a bit where it is on Grafana dashboard. I had a really hard time understanding what the Instant switch actually means